packages/gatsby-theme-api-docs/src/utils/render-regex-property.js

```diff
@@ -0,0 +1,28 @@
+import { css } from '@emotion/react';
+import { Markdown, designSystem } from '@commercetools-docs/ui-kit';
+
+function renderRegexProperties(data) {
```

Comment:
I'm not the react conventions guru here but to my understanding this is not a utility but a component, i.e. would rather live somewhere in the components folder and would rather have an export of the component name instad  of a `renderFooBar` naming